test/clj_sqs_extended/core_test.clj

```diff
@@ -37,19 +45,22 @@
   (testing "Receiving multiple messages from FIFO queue in correct order"
     (let [client (fixtures/localstack-sqs)
           url (fixtures/test-fifo-queue-url)]
-      (sqs-ext/send-fifo-message client url test-message-1 "red")
-      (sqs-ext/send-fifo-message client url test-message-2 "blue")
-      (let [message (sqs-ext/receive-message client url)]
-        (is (= test-message-1 (:body message))))
-      (let [message (sqs-ext/receive-message client url)]
-        (is (= test-message-2 (:body message)))))))
+      (doseq [format [:transit :json]]
+        (let [group-id-1 (str format "-1")
+              group-id-2 (str format "-2")]
+          (sqs-ext/send-fifo-message client url test-message-1 group-id-1 {:format format})
+          (sqs-ext/send-fifo-message client url test-message-2 group-id-2 {:format format}))
+        (let [message (sqs-ext/receive-message client url {:format format})]
+          (is (= test-message-1 (:body message))))
+        (let [message (sqs-ext/receive-message client url {:format format})]
+          (is (= test-message-2 (:body message))))))))
```

Comment:
are 2 messages sufficient entropy to guarantee that the ordering works?

Reply:
Extended to 5.